gpt-survey-summarizer/Chart.yaml

```diff
@@ -19,6 +19,9 @@ dependencies:
   - name: redis
     version: "19.1.0"
     repository: "https://charts.bitnami.com/bitnami"
+  - name: redisinsight
+    version: "0.4.5"
+    repository: "https://heywood8.github.io/helm-charts"
```

Comment:
**question:** What is the benefits using the [`heywood8/redisinsight`](https://artifacthub.io/packages/helm/heywood8-helm-charts/redisinsight) over [`hansehe/redisinsight`](https://artifacthub.io/packages/helm/redisinsight-helm/redisinsight)?

I found that this chart:
- have not been updated for a long time
- is running an old version of the RedisInsight application (2.28)
- is deploying a non-official [docker image](https://hub.docker.com/r/heywood8/redisinsight/tags)
- is not compatible with the official [docker image](https://hub.docker.com/r/redis/redisinsight/tags)
- does not allow us to deploy the [latest version of the application](https://hub.docker.com/layers/redis/redisinsight/2.48.0/images/sha256-013724efdbe890165a60bb5668503fa2b25d4e418d9f9bc04915e9086d0a92f3?context=explore)

I suggest to use the chart from `hansehe`, or, write our own if it is not matching exactly our need. What do you think?

Reply:
I tried [hansehe/redisinsight](https://artifacthub.io/packages/helm/redisinsight-helm/redisinsight) now for the 3rd time, and I am still unable to start it. 

It seems like it has a permission issue and cannot create the logs directory. 

```
❯ k logs redisinsight-0
Running docker-entry.sh
/usr/src/app/redisinsight/api/node_modules/file-stream-rotator/lib/helper.js:16
            throw error;
            ^

Error: EACCES: permission denied, mkdir '/data/logs'
    at Object.mkdirSync (node:fs:1398:3)
    at makeDirectory (/usr/src/app/redisinsight/api/node_modules/file-stream-rotator/lib/helper.js:12:12)
    at FileStreamRotator.createNewLog (/usr/src/app/redisinsight/api/node_modules/file-stream-rotator/lib/FileStreamRotator.js:140:36)
    at FileStreamRotator.rotate (/usr/src/app/redisinsight/api/node_modules/file-stream-rotator/lib/FileStreamRotator.js:133:14)
    at new FileStreamRotator (/usr/src/app/redisinsight/api/node_modules/file-stream-rotator/lib/FileStreamRotator.js:22:14)
    at Object.getStream (/usr/src/app/redisinsight/api/node_modules/file-stream-rotator/lib/index.js:32:12)
    at new DailyRotateFile (/usr/src/app/redisinsight/api/node_modules/winston-daily-rotate-file/daily-rotate-file.js:80:57)
    at Object.<anonymous> (/usr/src/app/redisinsight/api/dist/config/logger.js:18:27)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10) {
  errno: -13,
  syscall: 'mkdir',
  code: 'EACCES',
  path: '/data/logs'
}

Node.js v18.18.2
```

So I would say that both charts have their problems. We can hack it together to make it work, but creating a quick chart would be the best probably if we want to host this.